dbus/methods.go

```diff
@@ -288,6 +288,15 @@ func (c *Conn) listUnitsInternal(f storeFunc) ([]UnitStatus, error) {
 	return status, nil
 }
 
+// GetUnitByPID returns an array with all currently loaded units. Note that
```

Comment:
There is part of a sentence missing after "Note that".

Reply:
ack